packages/react/src/components/Popup/Popup.tsx

```diff
@@ -439,8 +438,12 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
     const popupWrapperAttributes = {
       ...(rtl && { dir: 'rtl' }),
       ...accessibility.attributes.popup,
-      ...accessibility.keyHandlers.popup,
-
+      onKeyDown: (e: React.KeyboardEvent) => {
```

Comment:
I know it is too late now, but I was just wondering whether our suggestion for the users would have been override the `onKeyDown` for the Popup and stop propagating if you want? This seem too much specific to the client's needs that we had at that moment.. @layershifter @sophieH29